lib/bgpstream_elem.h

```diff
@@ -107,6 +107,19 @@ typedef enum {
 
 } bgpstream_elem_type_t;
 
+typedef struct struct_bgpstream_annotations_t {
+  
+  /** RPKI active */
```

Comment:
minor issue: this annotation struct seems to be only about `rpki` annotation. if we want to keep it generic, we can keep the struct name, and enclose rpki-related fields in #ifdef block. @alistairking

Reply:
Agreed.

Reply:
We already had the same problem the other way around in the last pull request.

https://github.com/CAIDA/libbgpstream/pull/26/commits/99e682f7170c8786ce42386c39664c9729c41564 - "2. we always include all the possible fields in the public structures. this is much better since the API will be stable no matter what compile time options are used."

Reply:
hah yeah, you're right. thanks for being more awake than me. this should stay as-is.